CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # nf-core/methylseq
 
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
```

Comment:
Add in a header here for `dev` updates to the pipeline and something suggesting that you have updated to tools v1.9 e.g. https://github.com/nf-core/atacseq/blob/dev/CHANGELOG.md#unpublished-version--dev

Reply:
Actually wait. Release code has been sitting on `dev` 🤦‍♂ Could still add the update in the existing section.